plugin/src/main/java/com/craftmend/openaudiomc/generic/redis/packets/adapter/RedisTypeAdapter.java

```diff
@@ -28,6 +28,7 @@ public OARedisPacket deserialize(JsonElement json, Type typeOfT, JsonDeserializa
         JsonObject jsonObject = json.getAsJsonObject();
         String type = jsonObject.get("type").getAsString();
         String senderUuid = jsonObject.get("senderUuid").getAsString();
+        //
         JsonElement element = jsonObject.get("payload");
```

Comment:
but when removing the `payload` in the message, of course on the other end it is not found and causes NullPointerException later